api/src/test/java/org/openmrs/api/db/hibernate/HibernateAdministrationDAOTest.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api.db.hibernate;
+
+import org.openmrs.api.context.Context;
+import org.openmrs.api.db.AdministrationDAO;
+import org.hibernate.SessionFactory;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Role;
+import org.openmrs.test.BaseContextSensitiveTest;
+import org.openmrs.test.Verifies;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.validation.BindException;
+import org.springframework.validation.Errors;
+
+public class HibernateAdministrationDAOTest extends BaseContextSensitiveTest {
+
+	@Autowired
+	private HibernateAdministrationDAO dao;
+
+	private SessionFactory sessionFactory;
+
```

Comment:
Can you please have a test with a field where the column name differs from its mapped java property name? E.g Location.cityVillage